doc/en/api/1.0.0/styles.yaml

```diff
@@ -85,36 +102,23 @@ paths:
         
         The style definition body can be requested by either appending the file extension of the style file (e.g., "{style}.sld" or "{style}.css") or by specifying the correct media type for the style definition in the "Accept" header. Below are common style formats and the corresponding media types that can be used in the Accept header to request the style definition body.
         
-        - application/vnd.ogc.sld+xml for SLDs
+        - application/vnd.ogc.sld+xml for SLD 1.0.0 SLDs
+        - application/vnd.ogc.se+xml for SLD 1.1.0 SLDs
         - application/vnd.geoserver.geocss+css for css styles
         - application/vnd.geoserver.ysld+yaml for ysld styles
         - application/vnd.geoserver.mbstyle+json for mb styles
       produces:
         - application/xml
         - application/json
         - application/vnd.ogc.sld+xml
+        - application/vnd.ogc.se+xml
         - text/html
       parameters:
         - name: style
           in: path
           required: true
           description: Name of the style to retrieve.
           type: string
-        - name: purge
-          in: query
-          required: false
-          description: Specifies whether the underlying file containing the style should be deleted on disk. Allowable values are "true" or "false" (default).
-          type: boolean
-        - name: quietOnNotFound 
-          in: query
-          required: false
-          description: When set to "true", will not log an exception when the style is not present. The 404 status code will still be returned. Allowable values are "true" or "false" (default).
-          type: boolean
```

Comment:
I thought this was handled by the dispatcher, so is valid everywhere ...

Reply:
Yep, but we've been super inconsistent documenting it. I'm going to do a pass where I add it everywhere at the end. One thing I might do is add it as a definition in a separate file, and ref it everywhere.

Reply:
Can we remove it from everywhere because it does not add value to the client / server relationship. It is kind of a detail added for the automated test cases of interest to nobody.

So if you have to we can add a "note" when we introduce the REST API and take it out of the swagger docs.